packages/client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import IconButton from '@material-ui/core/IconButton';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import PropTypes from 'prop-types';
+import Modal from '../Modal';
+import Wizard from '../Wizard';
+import Button from '../Button';
+import styled from 'styled-components';
+import { COLORS } from '../../constants';
+import { withStyles } from "@material-ui/styles";
+
+
+const Wiz_content = ['page1', <Button>oh boi</Button>, 'page3']
+
+const TextThemes = {
+  fontSize: {
+    default: 18,
+    lg: 22,
+  },
+  fontWeight: {
+    default: 'normal',
+    lg: 'bold',
+  }
+}
+
+const LinkStyled = styled(Link)`
+  font-family: Muli, sans-serif;
+  font-color: ${COLORS.blue};
+  padding-right: 40px;
+  font-size: ${(props) => TextThemes.fontSize[props.ver]}px;
+  font-weight: ${(props) => TextThemes.fontWeight[props.ver]};
+  text-decoration: none;
+
+  &:link {
+    color: ${COLORS.blue};
+  }
+  &:visited {
+    color: ${COLORS.blue};
+  }
+  &:hover {
+    color: ${COLORS.yellow};
+    text-decoration: none;
+  }
+  &:active {
+    color: ${COLORS.yellow};
+    text-decoration: none;
+  }
+`
+
+export default function NavBar(props)  {
+
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  let menuOpen = false;
+
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    menuOpen = false;
+    setAnchorEl(null);
+  };
+
+  let userLinks;
+  if (false) {
+    userLinks = (
+      <>
+        <LinkStyled href='/login' ver='default'>Login</LinkStyled>
+        <div/>
+        <Modal title="Sign Up" trigger={<Button theme='accent' size='smmd'> Sign Up </Button>}> <Wizard content={Wiz_content} /> </Modal>
+      </>
+    );
+  } else {
+    userLinks = (
+      <>
+        <IconButton
+          aria-label="account of current user"
+          aria-controls="menu-navbar"
+          aria-haspopup="true"
+          onClick={handleMenu}
+          color="inherit"
+          onMouseOver={handleMenu}
+        >
+          <AccountCircleIcon/>
+          <div style={{ padding:'10px' }}/>
+          <LinkStyled ver='default' style={{ color: COLORS.blue }}>
+            {'Tim Beaver'}
+          </LinkStyled>
+        </IconButton>
+        <Menu
```

Comment:
opening the dropdown seems to disable scrolling/remove scrollbar on the rest of the page, not sure what might be causing that. maybe not a big deal though

![image](https://user-images.githubusercontent.com/12561777/88492523-fb711380-cf70-11ea-9a44-a011bcb41ed5.png)

![image](https://user-images.githubusercontent.com/12561777/88492527-04fa7b80-cf71-11ea-9875-e6a6dbd802e8.png)

Reply:
Apparently the menu dropdown creates several non-visible overlays when it opens? (mentioned in a post when I was debugging a different issue). I'm not sure if that would be related to this problem?